ListingManager/ListingInformation.cs

```diff
@@ -1,21 +1,26 @@
 using System;
+using System.Collections.Generic;
 using System.Text.RegularExpressions;
 
 namespace ListingManager
 {
-    public class ListingInformation
+    public partial class ListingInformation
     {
+        public static List<string> approvedFileTypes = new() { ".cs", ".xml" };
```

Comment:
I would avoid the public field and instead make it a property.
```suggestion
        public static IReadOnlyList<string> ApprovedFileTypes { get; } = new[] { ".cs", ".xml" };
```

Reply:
Fixed